assets/src/js/accordion/toggler.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Accordion Item open/close Triggers
+ */
+
+/**
+ * Find an ancestor element with a particular class.
+ *
+ * @param {Element} element Element to start search at.
+ * @param {string} selector Class name to search for.
+ * @returns {*} Ancestor element.
+ */
+const findAncestor = ( element, selector ) => {
```

Comment:
I don't think we have to support Internet Explorer here, so we can use [`Element.closest`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest).